core/src/main/java/org/commonjava/indy/core/inject/AbstractNotFoundCache.java

```diff
@@ -0,0 +1,30 @@
+package org.commonjava.indy.core.inject;
+
+import org.commonjava.indy.model.core.StoreKey;
+import org.commonjava.maven.galley.model.Location;
+import org.commonjava.maven.galley.spi.nfc.NotFoundCache;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by ruhan on 12/1/17.
+ */
+public abstract class AbstractNotFoundCache implements NotFoundCache
```

Comment:
Should this be in galley? It seems like it belongs with the general NFC functionality...

Should the methods be added to NotFoundCache? IMO that would be a good idea, just so we can continue to code to an interface.

Reply:
See my point above. But I have no strong opinion here.